### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'getAttribute')

### DISCUSSION THREAD

**Discussion by realamirhe** [sampotts/plyr#2599] [Q&A]
https://github.com/sampotts/plyr/blob/32444c9851414a1f1947add04e4c3be47b964b0e/src/js/plugins/youtube.js#L109-L115

If `player.media` has been set to `null` (happen in the next SSR offloading)
the `currentId` will be null and the `!is.empty(currentId)` will be `false` so we will face an issue in accessing the `getAttribute` in the next line `player.media.getAttribute('src')`

Is this expected behavior? 
Is it guaranteed that media property to have value always?

**Comment by realamirhe** [comment]
Any update or clarification :) ?

**Reply by kasvith** [reply]
im having the same issue
